irohad/ametsuchi/impl/postgres_specific_query_executor.cpp

```diff
@@ -484,10 +484,10 @@ namespace iroha {
                ? R"(, base_row AS(SELECT row FROM my_txs WHERE hash = lower(:hash) LIMIT 1))"
                : ""),
           (first_hash ? R"(JOIN base_row ON my_txs.row >= base_row.row)" : ""),
-          "AND (:first_tx_time::text IS NULL OR :first_tx_time<ts)",
-          "AND (:last_tx_time::text IS NULL OR :last_tx_time>ts )",
-          "AND (:first_tx_height::text IS NULL OR :first_tx_height<height)",
-          "AND (:last_tx_height::text IS NULL OR :last_tx_height>height )");
+          "AND (:first_tx_time::text IS NULL OR :first_tx_time<=ts)",
```

Comment:
this was in other PR..

Reply:
It still not in 1.2 and tests of this PR are dependent from it. This chacnge can be in both PR. Who will be the first - than it will make it.